docs/theia.md

```diff
@@ -0,0 +1,68 @@
+# Theia
+
+Theia is the command-line tool which provides access to Theia network flow
+visibility capabilities.
+
+## Table of Contents
+
+<!-- toc -->
+- [Installation](#installation)
+- [Compile](#compile)
+- [Usage](#usage)
+<!-- /toc -->
+
+## Installation
+
+Starting with Theia release v0.1, we publish the theia binaries for
+different OS / CPU Architecture combinations. Head to the [releases
+page](https://github.com/antrea-io/theia/releases) and download the
+appropriate one for your machine. For example:
+
+On Mac & Linux:
+
+```bash
+curl -Lo ./theia "https://github.com/antrea-io/theia/releases/download/<TAG>/theia-$(uname)-x86_64"
+chmod +x ./theia
+mv ./theia /some-dir-in-your-PATH/theia
+theia help
+```
+
+For Linux, we also publish binaries for Arm-based systems.
```

Comment:
It is a single binary? Then: "binary"

Reply:
Take a look at antctl, there will be two binaries for arm and arm64 binaries.

Reply:
Imho this statement is probably misplaced here. Here we expect instructions for fetching an installing the CLI. This note about ARM support can be in the paragraph at lines 16-18